core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -120,7 +121,7 @@
    * @param blockIds a list of block ids to remove from Alluxio space
    * @param delete whether to delete blocks' metadata in Master
    */
-  void removeBlocks(List<Long> blockIds, boolean delete) throws UnavailableException;
+  void removeBlocks(Collection<Long> blockIds, boolean delete) throws UnavailableException;
```

Comment:
Changed signature so the ConcurrentLinkedQueue from DefaultDeletionContext can be used. After all the caller just needs something to iterate.